src/WebUI/WorkLoads/StatefulSetTable.tsx

```diff
@@ -142,7 +147,7 @@ export class StatefulSetTable extends BaseComponent<IStatefulSetTableProperties,
         let podString: string = "";
         if (statefulSet.status.replicas > 0) {
             statusProps = Utils._getPodsStatusProps(statefulSet.status.currentReplicas, statefulSet.status.replicas);
-            podString = format("{0}/{1}", statefulSet.status.currentReplicas, statefulSet.status.replicas);
+            podString = localeFormat("{0}/{1}", statefulSet.status.currentReplicas, statefulSet.status.replicas);
```

Comment:
Strings like this are not locale-aware. Is there any reason we should not use format/ use localeFormat in such cases?

Reply:
these are user facing, we should use localeFormat, so if any changes are taken care automatically.